custom/ucr_ext/tests.py

```diff
@@ -379,3 +379,296 @@ def test_form_filter(self):
             }
         })
         self.assertEqual(2, expression({"some_field", "some_value"}, context))
+
+
+class TestGetCaseHistoryExpressionTest(TestCase):
+
+    def setUp(self):
+        self.case = {
+            '_id': uuid.uuid4().hex,
+            'opened_on': datetime(2014, 12, 10),
+            'modified_on': datetime(2015, 3, 10),
+            'type': 'test_case',
+            'actions': [
+                {
+                    'action_type': 'update',
+                    'date': datetime(2015, 1, 10),
+                    'updated_unknown_properties': {
+                        'foo': 'a'
+                    },
+                },
+                {
+                    'action_type': 'update',
+                    'date': datetime(2015, 1, 11),
+                    'updated_unknown_properties': {
+                        'foo': 'a'
+                    },
+                },
+                {
+                    'action_type': 'update',
+                    'date': datetime(2015, 1, 12),
+                    'updated_unknown_properties': {
+                        'foo': 'b'
+                    },
+                },
+                {
+                    'action_type': 'update',
+                    'date': datetime(2015, 2, 3),
+                    'updated_unknown_properties': {
+                        'foo': 'b'
+                    },
+                },
+                {
+                    'action_type': 'update',
+                    'date': datetime(2015, 3, 4),
+                    'updated_unknown_properties': {
+                        'foo': 'c'
+                    },
+                },
+            ]
+        }

```

Comment:
The JSON representation of SQL case actions is different and doesn't include any of the properties that changed. The SQL one only has `date`,`server_date`, `xform_id`, `sync_log_id`.